config.default

```diff
@@ -286,11 +286,7 @@ movingAveragePoints = 12
 # compare to observations?
 compareWithObservations = True
 # list of region indices to plot from the region list in [regions] below
-regionIndicesToPlot = [6]
-# Number of points over which to compute moving average (e.g., for monthly
-# output, movingAveragePoints=12 corresponds to a 12-month moving average
-# window)
-movingAveragePoints = 12
+regionIndex = 5
```

Comment:
Do you mean the rename of the config section?
I see your point. I have the same for the MOC. I think it should be OK as long as we add instructions in the config.default. But I will let @xylar comment on this too.